Linear_cell_complex/include/CGAL/draw_linear_cell_complex.h

```diff
@@ -1,425 +1,298 @@
-// Copyright (c) 2018 CNRS and LIRIS' Establishments (France).
+// Copyright (c) 2022 GeometryFactory Sarl (France).
```

Comment:
Please don't modify anything in the header (except your name as author)

Reply:
Backed to the original.